marklogic-data-hub/src/main/java/com/marklogic/hub/dataservices/StepService.java

```diff
@@ -140,21 +147,30 @@ private void deleteStep(BaseProxy.DBFunctionRequest request, String stepDefiniti
    */
     com.fasterxml.jackson.databind.JsonNode getStepsByType(Stream<String> propertiesToReturn);
 
+    /**
+     * Invokes the saveStep operation on the database server
+     *
+     * @param stepDefinitionType	provides input
+     * @param stepProperties	provides input
+     * @return	Return the created/updated step document
+     */
+    com.fasterxml.jackson.databind.JsonNode saveStep(String stepDefinitionType, com.fasterxml.jackson.databind.JsonNode stepProperties);
```

Comment:
Shouldn't DS have removed this method? I think we only want one saveStep method, and I think it's good that the client has to explicitly choose whether to do an overwrite or not.

Reply:
I just tried this locally - I added overwrite:boolean to the API, then ran "gradle generatestepinterface", and the method was updated to include the boolean. That's what we want, we don't want a saveStep method that doesn't require the boolean.